src/tokens-studio/spectrum2-colors/spectrum2/alias/dark.json

```diff
@@ -761,7 +761,7 @@
           }
         },
         "subtle-default": {
-          "value": "{Palette.gray.100}",
+          "value": "{Palette.gray.300}",
```

Comment:
@larz0 In the [token specs](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=25369-7932), I'm seeing the change documented was for 'neutral-**subdued**-background-color-default' and not 'neutral-**subtle**-background-color-default'

I think what you have here is correct, because neutral-subdued has a different usage but wanted to double-check that this is confirmed. I'm leaving a comment directly in the token spec file too for Miruna!

Reply:
@lynnhao yeah I think it's a typo because the values she referred to in S1 are for subtle and not subdued.